tools/python/brownie/great_ape_safe/ape_api/across.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from helpers.addresses import registry, r
+
+
+
+class Across:
+    def __init__(self, safe):
+        self.safe = safe
+
+        # See https://docs.across.to/v/developer-docs/developers/across-api#calculating-suggested-fees
+        # Any chain's token address can be used
+        self.api_url = "https://across.to/api/suggested-fees"
+        self.spokepool = self.safe.contract(r.across.spoke_pool)
+
+
+    def get_token_bridge_info(self, token_address, dest_chain_id, amount):
+        # https://github.com/anyswap/CrossChain-Router/wiki/How-to-integrate-AnySwap-Router
+        params = {"destinationChainId": dest_chain_id, "token": token_address, "amount": amount}
```

Comment:
Per comment above, it's advisable to specify the originChainId, though it should work OK without it in most cases.